packages/expo-router/src/typed-routes/__tests__/fixtures/default.d.ts

```diff
@@ -11,7 +11,7 @@ declare module 'expo-router' {
+      href: Router.RelativePathString | Router.ExternalPathString | `/_sitemap${`?${string}` | `#${string}` | ''}` | `/apple${`?${string}` | `#${string}` | ''}` | `/banana${`?${string}` | `#${string}` | ''}` | `${'/(group)'}/static${`?${string}` | `#${string}` | ''}` | `/static${`?${string}` | `#${string}` | ''}` | `${'/(group)'}${`?${string}` | `#${string}` | ''}` | `/${`?${string}` | `#${string}` | ''}` | `${'/(group)'}${'/(a)' | '/(b)'}/folder${`?${string}` | `#${string}` | ''}` | `/folder${`?${string}` | `#${string}` | ''}` | { pathname: Router.RelativePathString, params?: Router.UnknownInputParams } | { pathname: Router.ExternalPathString, params?: Router.UnknownInputParams } | { pathname: `/_sitemap`; params?: Router.UnknownInputParams; } | { pathname: `/apple`; params?: Router.UnknownInputParams; } | { pathname: `/banana`; params?: Router.UnknownInputParams; } | { pathname: `${'/(group)'}/static` | `/static`; params?: Router.UnknownInputParams; } | { pathname: `${'/(group)'}` | `/`; params?: Router.UnknownInputParams; } | { pathname: `${'/(group)'}${'/(a)' | '/(b)'}/folder` | `/folder`; params?: Router.UnknownInputParams; } | `/colors/${Router.SingleRoutePart<T>}${`?${string}` | `#${string}` | ''}` | `/animals/${string}${`?${string}` | `#${string}` | ''}` | `/mix/${Router.SingleRoutePart<T>}/${Router.SingleRoutePart<T>}/${string}${`?${string}` | `#${string}` | ''}` | `${'/(group)'}${'/(a)' | '/(b)'}/folder/${Router.SingleRoutePart<T>}${`?${string}` | `#${string}` | ''}` | `/folder/${Router.SingleRoutePart<T>}${`?${string}` | `#${string}` | ''}` | `${'/(group)'}${'/(a)' | '/(b)'}/folder/${string}${`?${string}` | `#${string}` | ''}` | `/folder/${string}${`?${string}` | `#${string}` | ''}` | `/(c)/folder/${Router.SingleRoutePart<T>}${`?${string}` | `#${string}` | ''}` | `/folder/${Router.SingleRoutePart<T>}${`?${string}` | `#${string}` | ''}` | { pathname: `/colors/[color]`, params: Router.UnknownInputParams & { color: string | number; } } | { pathname: `/animals/[...animal]`, params: Router.UnknownInputParams & { animal: (string | number)[]; } } | { pathname: `/mix/[fruit]/[color]/[...animals]`, params: Router.UnknownInputParams & { fruit: string | number;color: string | number;animals: (string | number)[]; } } | { pathname: `${'/(group)'}${'/(a)' | '/(b)'}/folder/[slug]` | `/folder/[slug]`, params: Router.UnknownInputParams & { slug: string | number; } } | { pathname: `${'/(group)'}${'/(a)' | '/(b)'}/folder/[...slug]` | `/folder/[...slug]`, params: Router.UnknownInputParams & { slug: (string | number)[]; } } | { pathname: `/(c)/folder/[slug]` | `/folder/[slug]`, params: Router.UnknownInputParams & { slug: string | number; } };
```

Comment:
This test seem seems to have been changed? What was the error here, maybe these test need to be excluded from eslint

Reply:
sorry, not really sure how these got changed. reverted

Reply:
I had the same changes just by running "yarn test" in packages in packages/expo-router